src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs

```diff
@@ -1221,6 +1222,16 @@ public Message Request(Message message, TimeSpan timeout)
             return (Message)Call(message.Headers.Action, false, operation, new object[] { message }, Array.Empty<object>(), timeout);
         }
 
+        public virtual Task<Message> RequestAsync(Message message)
```

Comment:
Good point -- will remove virtual.  Don't remember why I did it in the first place.

---
In reply to: [78485372](https://github.com/dotnet/wcf/pull/1515#discussion_r78485372) [](ancestors = 78485372)